omod/src/main/webapp/admin/concepts/conceptDrugForm.jsp

```diff
@@ -10,9 +10,8 @@
 <script type="text/javascript">
 	dojo.require("dojo.widget.openmrs.ConceptSearch");
 	dojo.require("dojo.widget.openmrs.OpenmrsPopup");
-
 	<request:existsParameter name="autoJump">
-		var autoJump = <request:parameter name="autoJump"/>;
+		var autoJump = ${openmrs:getSafeJsBoolean(param.autoJump)};
```

Comment:
I tried that, but I found that <c:out> does some html-encoding that will cause the JS to break for certain inputs. Since the value is injected straight into JS code (and not HTML) we really need a JS-specific filtering function.